werkomgeving/gegeven/index.md

```diff
@@ -63,21 +63,21 @@ Ook kunnen er gegevens opgenomen worden in het gegevensobject over andere object
 
 ## Een gegevensobjecttype
 
-Een gegevensobjecttype is *een typering van gelijksoortige gegevensobjecten*. Met een gegevensobjecttype beschrijven we hoe een afzonderlijk gegevensobject er uit ziet, als groepering van gegevens. Van een gegevensobjecttype wil je de volgende zaken weten:
+Een gegevensobjecttype is *een typering van gelijksoortige gegevensobjecten*. Met een gegevensobjecttype beschrijven we hoe een afzonderlijk gegevensobject er uit ziet, als groepering van gegevens. Van een gegevensobjecttype wil je de volgende zaken weten: 
 
 1. Welke gegevenstypen zijn onderdeel van dit gegevensobjecttype?
-2. Welke gegevenstypen zijn beoogd voor de identificatie van het object waarover de gegevens gaat? (dwz: welke gegevenstypen vormen de sleutel)
+2. Welke van die gegevenstypen zijn beoogd voor de identificatie van het object waarover de gegevens gaan? (dwz: welke gegevenstypen vormen de sleutel)
 3. Welke metagegevenstypen worden toegevoegd aan het gegevensobjecttype?
-4. Welke gegevenstypen gaan over andere objecten dan het gegevensobjecttype zelf? En hoe is het toegangspad van dit object naar het object waarover deze objecttypen gaat? (zoals in het «naam van de woonplaats» voorbeeld: dit gegevenstype gaat niet over het hoofdonderwerp [Jan] maar over zijn woonplaats)
-5. (Optioneel - relatie naar het CIM): welk objecttype is het *hoofdonderwerp* van dit gegevensobjecttype, dwz: over welk type objecten *gaan* gegevensobjecten van dit gegevensobjecttype?
```

Comment:
Wat bedoel je hier precies mee?

Reply:
hoofdonderwerp is in het andere document verder uitgewerkt.